<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicFlow ‚Äî –ê–≤—Ç–æ—Ä–∏–∑–∞—Ü–∏—è</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="spacer_top">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="login_visit_text">
     <property name="text">
      <string>–î–æ–±—Ä–æ –ø–æ–∂–∞–ª–æ–≤–∞—Ç—å –≤ MusicFlow üéµ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="login_table">
     <property name="placeholderText">
      <string>–õ–æ–≥–∏–Ω</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_table">
     <property name="placeholderText">
      <string>–ü–∞—Ä–æ–ª—å</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="login_button">
       <property name="text">
        <string>–í–æ–π—Ç–∏</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="go_to_register">
       <property name="text">
        <string>–†–µ–≥–∏—Å—Ç—Ä–∞—Ü–∏—è</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="login_error_table">
     <property name="text">
      <string></string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #EF4444;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #121212;
    color: #FFFFFF;
    font-family: "Segoe UI", "Roboto", sans-serif;
    font-size: 14px;
}

QLineEdit {
    background-color: #1E1E1E;
    border: 2px solid #333;
    border-radius: 8px;
    padding: 6px;
    color: #FFFFFF;
}

QLineEdit:focus {
    border: 2px solid #3B82F6;
}

QPushButton {
    background-color: #3B82F6;
    border-radius: 8px;
    padding: 6px;
    color: white;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2563EB;
}

QPushButton:pressed {
    background-color: #1D4ED8;
}
   </string>
  </property>
 </widget>
 <resources/>
 <connections/>
</ui>
